kubeshop/monokle
# feat: telemetry modal

## Changes

- adds a notification for users who have not interfered with telemetry data

## Fixes

- fix: #1512 

## How to test it

- if you've played with the telemetry data just add these somewhere in the code the on the next reload the telemetry modal should show
`
electronStore.delete('appConfig.disableEventTracking');
electronStore.delete('appConfig.disableErrorReporting');
`

## Screenshots

- https://user-images.githubusercontent.com/5509112/161067277-58fad75a-d29d-45a4-bfef-c84201efb1c2.mov

## Checklist
